Tools/cases_generator/generate_cases.py

```diff
@@ -242,6 +252,7 @@ class InstructionFlags:
     HAS_CONST_FLAG: bool
     HAS_NAME_FLAG: bool
     HAS_JUMP_FLAG: bool
+    IS_UOP_FLAG: bool = False  # Set later
```

Comment:
I don't think this is the right way to frame this.
I think it is better to regard all instructions as a list of uops. Many of those lists have length 1, but there is no need to special case it.

Reply:
Agreed. I have such a design worked out in my head.

Reply:
Done.